### PR TITLE
[ntfs2gtfs] write stops.txt

### PR DESCRIPTION
**Pull request by datanel**
see https://github.com/CanalTP/navitia_model/blob/master/src/documentation/ntfs2gtfs.md#stopstxt

### CODE REVIEW

File: src/gtfs/mod.rs [hunk 1]
```diff
@@ -47,25 +48,102 @@ struct Agency {
...
+            lat: obj.coord.lat,
+            lon: obj.coord.lon,
+            fare_zone_id: None,
+            location_type: StopLocationType::StopArea,
+            parent_station: None,
+            code: None,
+            desc: "".to_string(),
+            wheelchair_boarding: None,
+            url: None,
+            timezone: None,
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: You forgot timezone

File: src/gtfs/mod.rs [hunk 2]
```diff
@@ -47,25 +48,102 @@ struct Agency {
...
+    fn from(obj: &objects::StopArea) -> Stop {
+        Stop {
+            id: obj.id.clone(),
+            name: obj.name.clone(),
+            lat: obj.coord.lat,
+            lon: obj.coord.lon,
+            fare_zone_id: None,
+            location_type: StopLocationType::StopArea,
+            parent_station: None,
+            code: None,
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: you should take it from codes, no?

File: src/gtfs/mod.rs [hunk 3]
```diff
@@ -47,25 +48,102 @@ struct Agency {
...
+            lat: obj.coord.lat,
+            lon: obj.coord.lon,
+            fare_zone_id: obj.fare_zone_id.clone(),
+            location_type: StopLocationType::StopPoint,
+            parent_station: Some(obj.stop_area_id.clone()),
+            code: None,
+            desc: "".to_string(),
+            wheelchair_boarding: None,
+            url: None,
+            timezone: None,
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: You forgot timezone

File: src/gtfs/mod.rs [hunk 4]
```diff
@@ -47,25 +48,102 @@ struct Agency {
...
+    fn from(obj: &objects::StopPoint) -> Stop {
+        Stop {
+            id: obj.id.clone(),
+            name: obj.name.clone(),
+            lat: obj.coord.lat,
+            lon: obj.coord.lon,
+            fare_zone_id: obj.fare_zone_id.clone(),
+            location_type: StopLocationType::StopPoint,
+            parent_station: Some(obj.stop_area_id.clone()),
+            code: None,
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: code from codes?

File: src/gtfs/mod.rs (hunk 2 above)
Comment (prhod) [reply src/gtfs/mod.rs]: We do not read it for the moment. I've added this issue for this : https://github.com/CanalTP/navitia_model/issues/130

File: src/gtfs/mod.rs (hunk 4 above)
Comment (prhod) [reply src/gtfs/mod.rs]: same as above :p

**Pull request merged**